AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -68,12 +69,19 @@ - (void)_staticInitialize;
 
 @implementation ASDisplayNode
 
-// these dynamic properties all defined in ASLayoutOptionsPrivate.m
-@dynamic spacingAfter, spacingBefore, flexGrow, flexShrink, flexBasis,
-         alignSelf, ascender, descender, sizeRange, layoutPosition, layoutableType;
+// Dynamic properties for ASLayoutables
+@dynamic layoutableType, size;
+
+// Dynamic properties for sizing
+@dynamic width, height, minWidth, maxWidth, minHeight, maxHeight;
+@dynamic widthAsPoints, widthAsFraction, minWidthAsPoints, minWidthAsFraction, maxWidthAsPoints, maxWidthAsFraction;
```

Comment:
This are setters only.  If you try to use the getter you get an "unavailable" compiler warning.

I went back and forth on methods or these way I just think in Swift it feels way better to use a property instead of a method to set these values:

```Swift
layoutable.setWidthAsFraction(0.5)
layoutable.widthAsFraction = 0.5
```

Also I think it looks kind of out of place a bit :) e.g. 
```Swift
layoutable.flexGrow = YES
layoutable.flexShrink = NO
layoutable.setMinWidthAsPoints(300) // vs.
layoutable.minWidthAsPoints = 300
```